main/config.h

```diff
@@ -13,7 +13,7 @@
 #define IS_ATHENA 0
 #define IS_ATHENA_V2 0
 
-#define IS_SATELLITE 0
+#define IS_SATELLITE 1
```

Comment:
I don't think this should be default

Reply:
Sorry, the `main/config.h` should not have been modified. It should be fixed now